3.0/m_tgchange.cpp

```diff
@@ -0,0 +1,242 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2016 Adam <Adam@anope.org>
+ *   Copyright (C) 2020 Elizabeth Myers <elizabeth@interlinked.me>
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software: you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation, version 2.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* $ModAuthor: Adam */
+/* $ModAuthorMail: Adam@anope.org */
+/* $ModDesc: Implements "target change" detection to help prevent spam bots */
+/* $ModDepends: core 3 */
```

Comment:
This is technically still valid but the v3 style for build comments is to do this instead:

```suggestion
/// $ModDepends: core 3
```